docs/step-by-step/kubeapps-on-tkg/step-1.md

```diff
@@ -1,78 +1,95 @@
+![Retrieve app id and secret](./img/step-1-6.png)
+
+> **NOTE**: Store this file carefully as it contains important credentials which will be needed when configuring Pinniped and Kubeapps.
 
-> **TIP**: Any OIDC-compliant provider should expose a `.well-known/openid-configuration` ([example](https://console.cloud.vmware.com/csp/gateway/am/api/.well-known/openid-configuration)) endpoint where you can find other useful and required information. It will allow just using the base URL to discover the rest of the URLs (`authorization`, `token`, `end session`, `jwks` and `issuer`) automatically.
-> That is, for CSP, we will use henceforth this one: `https://console-stg.cloud.vmware.com/csp/gateway/am/api`.
+> **TIP**: Any OIDC-compliant provider should expose a `.well-known/openid-configuration` ([example](https://console.cloud.vmware.com/csp/gateway/am/api/.well-known/openid-configuration)) endpoint where you can find other useful and required information. This endpoint allows using the base URL to discover the rest of the URLs (`authorization`, `token`, `end session`, `jwks` and `issuer`) automatically. For CSP, the endpoint is `https://console-stg.cloud.vmware.com/csp/gateway/am/api`.
 
-### Make Pinniped Trust your OIDC Provider
+At this point, an OAuth2 application is configured.
 
-Once the OIDC provider has been fully configured, we need Pinniped to trust this provider, so that a successful authentication in the OIDC provider results in authentication in our TKG cluster.
+### Configure Pinniped to Trust the OIDC Provider
 
-Since Pinniped is already hiding the complexity of this process, we just need to add a `JWTAuthenticator` CustomResource in our cluster. To do so, simply edit the following excerpt accordingly and apply it to your TKG cluster.
+Once the OIDC provider has been fully configured, the next step is to configure Pinniped to trust this provider. This implies that a successful authentication with the OIDC provider results in authentication with the TKG cluster.
 
-> **TIP**: a look at [JWTAuthenticator official documentation](https://pinniped.dev/docs/howto/configure-concierge-jwt/) for further information.
+Since Pinniped manages this process, the only requirement is to a _JWTAuthenticator_ CustomResource in the cluster. To do so, follow the steps below:
 
-1. Create a file `kubeapps-jwt-authenticator.yaml` with the following content:
+> **TIP**: Learn more about [JWTAuthenticator](https://pinniped.dev/docs/howto/configure-concierge-jwt/).
+
+1. Create a file named `kubeapps-jwt-authenticator.yaml` with the following content. Replace the placeholders as follows:
+
+- Replace the  `OIDC-ISSUER-URL` with the _issuer_ URL of your OIDC provider. For CSP it is `https://console-stg.cloud.vmware.com/csp/gateway/am/api`.
+- Replace `CLIENT-ID` with the application ID obtained from the JSON file in the previous step.
+- Replace `EMAIL-ADDRESS` with TODO
```

Comment:
Pending this information

Reply:
Ahh no no, it was, indeed right

```yaml
...
  claims:
    username: email
```

(This is the data which will be requested to the identity provider, that is, the usermane will be the `email` field provided by CSP)

Reply:
Changed but not sure what I wrote is correct. Please check once.